diff --git a/node_modules/react-native-japanese-text-analyzer/android/build.gradle b/node_modules/react-native-japanese-text-analyzer/android/build.gradle
index 7ec958b..04b6dc7 100644
--- a/node_modules/react-native-japanese-text-analyzer/android/build.gradle
+++ b/node_modules/react-native-japanese-text-analyzer/android/build.gradle
@@ -19,10 +19,10 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
-    compileSdkVersion safeExtGet('JapaneseTextAnalyzer_compileSdkVersion', 29)
+    compileSdkVersion safeExtGet('JapaneseTextAnalyzer_compileSdkVersion', 30)
     defaultConfig {
         minSdkVersion safeExtGet('JapaneseTextAnalyzer_minSdkVersion', 16)
-        targetSdkVersion safeExtGet('JapaneseTextAnalyzer_targetSdkVersion', 29)
+        targetSdkVersion safeExtGet('JapaneseTextAnalyzer_targetSdkVersion', 30)
         versionCode 1
         versionName "1.0"
 
